src/components/ChangePlan/Checkout/Checkout.js

```diff
@@ -0,0 +1,90 @@
+import React, { useEffect, useState } from 'react';
+import { InjectAppServices } from '../../../services/pure-di';
+import HeaderSection from '../../shared/HeaderSection/HeaderSection';
+import { useIntl } from 'react-intl';
+import { Loading } from '../../Loading/Loading';
+import { Helmet } from 'react-helmet';
+import { ContactInformation } from './ContactInformation/ContactInformation';
+import { getCountries } from '../../../utils';
+
+const Checkout = ({ dependencies: { dopplerUserApiClient } }) => {
+  const intl = useIntl();
+  const _ = (id, values) => intl.formatMessage({ id: id }, values);
+  const selectOption = _('checkoutProcessForm.empty_option_select');
+
+  const checkoutSteps = {
+    contactInformation: 'contact-information',
+    billingInformarion: 'billing-information',
+    paymentInformarion: 'payment-information',
+  };
+
+  const [activeStep, setActiveStep] = useState(checkoutSteps.contactInformation);
+  const [state, setState] = useState({ loading: true });
+
+  const submitContactInformationForm = async (values) => {
+    const result = await dopplerUserApiClient.createOrUpdateContactInformation(values);
+
+    if (result.success) {
+      setActiveStep(checkoutSteps.contactInformation);
+    }
+
+    return result;
+  };
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const contactInformationResult = await dopplerUserApiClient.getContactInformationData();
+      const defaultValue = { key: '', value: selectOption };
+      const countries = [defaultValue, ...getCountries(intl.locale)];
```

Comment:
_defaultValue_ is not a country. Maybe it is the way to do it in React, but it sounds weird.

@jcamposmk what do you think about this pattern?

Reply:
Maybe you can add a new parameter with the default option in the `SelectFieldItem`?

```js
 export const SelectFieldItem = ({
   className,
   fieldName,
   label,
   type,
   placeholder,
   required,
+  defaultOption,
   values,
   ...rest
 }) => (
```